docs/the-new-architecture/use-app-template.md

```diff
@@ -1,7 +1,134 @@
+
+### Enable Hermes
+
+Hermes is an open-source JavaScript engine optimized for React Native. [Hermes will be the default engine in the future](https://github.com/reactwg/react-native-new-architecture/discussions/4), and we highly recommend you use it.
+
+Please [follow the instructions on the React Native website](hermes.md) in order to enable Hermes in your application.
+
+### Enable the New Architecture
+
+#### Target OS
+
+<Tabs groupId="platform" defaultValue={constants.defaultPlatform} values={constants.platforms} className="pill-tabs">
+<TabItem value="ios">
+
+Navigate to the `ios` directory and run the following:
+
+```shell
+# from `ios` directory
+RCT_NEW_ARCH_ENABLED=1 bundle exec pod install
+```
+
+Then build and run the app as usual:
+
+```shell
+npx react-native run-ios
+```
+
+> **Tip**: You need to re-run `pod install` each time a dependency with native
+> code changes. Make this command easier to run by adding it to `scripts` to your
+> project's `package.json` file:
+>
+> ```
+> "scripts": {
+>   "pod-install": "RCT_NEW_ARCH_ENABLED=1 bundle exec pod install"
+> }
+> ```
+>
+> and run it with `yarn pod-install`.
+
+#### Troubleshooting
+
+##### `react-native run-ios` fails
+
+If you see a build failure from `react-native run-ios`, there may be cached files from a previous build with
+the old architecture. Clean the build cache and try again:
+
+1. Open the project `ios/project.xcworkspace` in Xcode
+2. In XCode, choose Product > Clean Build Folder
+3. Try building and running the app again
+
+</TabItem>
+<TabItem value="android">
+
+Set the `newArchEnabled` property to `true` by **either**:
+
+- Changing the corresponding line in `android/gradle.properties`
+- Setting the environment variable `ORG_GRADLE_PROJECT_newArchEnabled=true`
+
+Then build and run the app as usual:
+
+```shell
+npx react-native run-android
+```
+
+> **Note**: You may notice longer build times with the New Architecture, due to
+> additional step of C++ compilation with the Android NDK. To improve your build
+> time, see
+> [Speeding Up Your Build Phase](docs/build-speed.md).
+
+</TabItem>
+</Tabs>
+
+### Confirming the New Architecture is in use
+
+After you build and run the app, when Metro serves the JavaScript bundle, you
+should see `"fabric": true` in the Metro logs:
+
+<img src="/docs/assets/metro-new-arch.png" alt="Metro shows fabric: true" width="600" />
+
+### Want to know more?
+
+If you'd like to view the code changes relevant for the New Architecture, take a
+look at the [upgrade
+helper from version 0.67.4 to 0.68.0](https://react-native-community.github.io/upgrade-helper/?from=0.67.4&to=0.68.0).
+Files that were added for the New Architecture are marked with a yellow banner.
+
+For further explanations of what each file is doing, check out these guides to walk through the changes step-by-step:
+
+#### Android
+
+- [Enabling TurboModules on Android](new-architecture-app-modules-android.md)
```

Comment:
I noticed the article in the sidebar is titled `Enabling TurboModule on Android` -- I've mostly seen `TurboModules`, plural, though. Is the plural correct?

Reply:
I think that in general, we refer to `TurboModules`. If we are referring to a specific one, we should use the singular `TurboModule`.